ts/util/html-utils.ts

```diff
@@ -68,14 +68,15 @@ export function addMisoIcon(container: HTMLElement, url: string) {
 }
 
 // all supported cell statuses
-export type CellStatus = "na" | "stopped" | "warning" | "error";
+export type CellStatus = "na" | "stopped" | "warning" | "error" | "paused";
 
 // map cell status to styles
 const highlightClasses: Record<CellStatus, string[]> = {
   na: ["text-grey-300", "bg-grey-150"],
   stopped: ["bg-grey-100"],
   warning: ["bg-yellow"],
   error: ["bg-red"],
+  paused: ["bg-grey-100"],
```

Comment:
Could use a single status for stopped and paused since the style is the same. Maybe rename to "disabled" for clarity